usr.sbin/chown/Makefile

```diff
@@ -3,9 +3,9 @@
 
 .include <src.opts.mk>
 
-PROG=	chown
-LINKS=	${BINDIR}/chown /usr/bin/chgrp
-MAN=	chgrp.1 chown.8
+PROG=	    chown
+SYMLINKS=	${BINDIR}/chown ${BINDIR}/chgrp
```

Comment:
We'd likely need to have this line as well
```
SYMLINKS+= ${BINDIR}/chown /usr/bin/chgrp
```
to ensure maximum compatibility. With that, I'd think the change would be fine

Reply:
Thinking more about it, I think the simplest thing would be to change LINKS to SYMLINKS and leave the directory where it is.